src/application.coffee

```diff
@@ -860,6 +869,7 @@ application.update = update = (force, scheduled = false) ->
 				else
 					updateStatus.state = UPDATE_IDLE
 			device.updateState(status: 'Idle')
+			updateStatus.lastFullUpdateCycle = process.hrtime()[0]
```

Comment:
Same here

Reply:
Same as above -  avoids problems with clock changes due to ntp